JPS_VIRTUALSENSOR/README.md

```diff
@@ -30,7 +30,7 @@ If you are not already logged in then, when prompted, enter the username and pas
 
 1) Initialise scope by:
 ```
-curl -X POST "http://localhost:3838/dispersion-interactor/InitialiseSimulation?ewkt=SRID=4326;POLYGON((-4.282264034358564%2050.26375198971232,-4.001705368451314%2050.26650880607838,-4.005497340234552%2050.44635115729881,-4.287117430213462%2050.44357678715814,-4.282264034358564%2050.26375198971232))&nx=400&ny=400"
+curl -X POST "http://localhost:3838/dispersion-interactor/InitialiseSimulation?ewkt=SRID=4326;POLYGON((7.58%2049.2,7.58%2049.236,7.61%2049.236,7.61%2049.2,7.58%2049.2))&nx=30&ny=30&citiesnamespace=pirmasensEPSG32633""
```

Comment:
I think I left this comment before but got lost, please don't overwrite the existing example

Reply:
I have changed it back to the original one.